tests/test_settings.py

```diff
@@ -20,9 +20,22 @@ def test_settings_public(rocket):
 
 
 def test_settings_oauth(logged_rocket):
+    # refresh is not done with any API call ever, so we need to call it manually here
+    logged_rocket.call_api_post(
+        "method.call/refreshOAuthService",
+        message='{"method": "refreshOAuthService", "params": []}',
+    )
     oauth_get = logged_rocket.settings_oauth().json()
     assert oauth_get.get("success")
-    assert not oauth_get.get("services")
+    if oauth_get.get("services"):
+        # remove the OAuth app Test beforehand, when this is not the first test run (for reproducibility)
+        logged_rocket.call_api_post(
```

Comment:
Is it possible to test this?

Reply:
as it is a normal requests.post, I check the requests.Response in c99c308. You think this suffices?

Reply:
Oh, I see now why codecov was complaining. Yes, that's enough.